package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "resin-cli",
   "version": "6.4.0",
-  "description": "Git Push to your devices",
+  "description": "The official Resin CLI tool",
```

Comment:
Indeed it should - updated.

We should also update the repo description to match. I've asked Apo to add me as an admin so I can do so, but feel free to do it yourself in the meantime, if you've got admin access.